hyperschedule/app.py

```diff
@@ -90,6 +92,33 @@ def view_api_v3():
         raise APIError("data not available yet")
     return api_response({"data": diff, "until": until, "full": full})
 
+@app.route("/upload-syllabus", methods = ['PUT', 'POST', 'OPTIONS'])
```

Comment:
This should only include `PUT` and `OPTIONS`. Remove `POST` from the list of methods.